### ISSUE TITLE
phylotree 17

### ISSUE THREAD

**Issue by leipzig** [MEEIBioinformaticsCenter/phy-mer#3]
Hey is there any way to generate a `Build_17_-_rCRS-based_haplogroup_motifs.csv`? I don't know how you made this but it would be a useful resource for the community.

**Comment by danielnavarrogomez** [comment]
Hi Lejemy,

This is on my To-Do list, I will try to get it tested and pushed by next month.

**Comment by leipzig** [comment]
any movement on this?

**Comment by danielnavarrogomez** [comment]
Trying to get all necessary data, sorry for the delay...

**Comment by danielnavarrogomez** [comment]
Hi @leipzig 

I'm working on a library that is able to extract info from PhyloTree, when done, I should be able to get the file equivalent file to Build_16_-_rCRS-based_haplogroup_motifs.csv but for the V17.

Check it out: https://github.com/MEEIBioinformaticsCenter/PhyloTree_lib

**Comment by leipzig** [comment]
ok thanks. I no longer work in this area but I will pass this along to my successor.

**Comment by xgai** [comment]
Fantastic,Daniel.
 Phy-met has been cited in multiple publications and it has been used more
and more.

Xiaowu

**Comment by nuin** [comment]
Hi

Any update on this? It seems that Phylotree_lib doesn't output the file format required by Phy-mer.

Thanks

**Comment by danielnavarrogomez** [comment]
Unfortunately, I only work on this project on my spare time... and looks like it's being a busy year.

**Comment by nuin** [comment]
Yes, the number of abandonware after publication in Bioinformatics is amazing. Thanks for letting me know.

**Comment by leipzig** [comment]
Hi Paulo,
Your donation check must have gotten lost in the mail. Could you provide a
valid credit card number?

Thanks!

**Comment by nuin** [comment]
Give me a call and I give you the number. 

1 - 800 - 328 - 5447

**Comment by danielnavarrogomez** [comment]
I'd rather prefer someone chipping in with time/commits than money :-)

**Comment by nuin** [comment]
I was going to, check the code for Phylotree_lib, but now I won't even use either anymore. 

You can thank @leipzig for that.
